### PR TITLE
Update README and start dev script

### PR DESCRIPTION
**Pull request by SpectreMercury**
- Update start dev script from `yarn dev` to `yarn dev:stie`
- Add automatically open browser tab after start dev server

### CODE REVIEW

**Review by Flouse — APPROVED** [review]

File: site/vite.config.ts [hunk 1]
```diff
@@ -17,4 +17,7 @@ export default defineConfig({
   ssr: {
     external: ['monaco-editor'],
   },
+  server: {
+    open: true,
+  },
```
Comment (Flouse) [inline site/vite.config.ts]:
```suggestion
```

Comment (Flouse) [reply site/vite.config.ts]: Try to fix the failed deployment:
image

**Review by oriole-pub — APPROVED** [review]

File: site/vite.config.ts (hunk 1 above)
Comment (SpectreMercury) [reply site/vite.config.ts]: OK~

Comment (SpectreMercury) [reply site/vite.config.ts]: image
Hi Flouse. In general, the turbo build command can sometimes time out or fail due to a variety of reasons, including network or CPU usage, as you mentioned.

One potential solution to this issue is to increase the timeout value using the --timeout flag. For example, you could try running turbo build index.html --timeout=60000 to increase the timeout to 60 seconds.

Comment (Flouse) [reply site/vite.config.ts]: ### The failed deployment
https://vercel.com/flouse/ckbacademy/E7zv79j6XjbsybyQo3BXqg6YhHhy#L27

**Pull request merged**